3rd_party/3rd_party.cmake

```diff
@@ -38,11 +38,11 @@ set(ARCH ${HOST_SYSTEM_PROCESSOR})
 set(CMAKE_SYSTEM_NAME ${HOST_SYSTEM_NAME})
 
 message(STATUS "CMAKE_SYSTEM_NAME = ${CMAKE_SYSTEM_NAME}")

```

Comment:
This line is _mega_ confusing when you read it in the build log, because it makes it look like a built in variable has the wrong value.